LEAF_Request_Portal/form.php

```diff
@@ -2487,6 +2488,23 @@ public function query($inQuery)
         $count = 0;
         foreach ($query['terms'] as $q)
         {
+            // Logic for AND/OR Operator's
+            $op = 'AND';
+            if ($q['op']) {
+                $op = $q['op'];
+            }
+            if ($count === 0) {
+                $op = '';
+                $conditions = '(';
+            }
+            if ($count != 0) {
```

Comment:
The logic here can be consolidated into count is zero or else.  don't need to check it again if it's just every other number but zero.

Reply:
Consolidated 👍